### ISSUE TITLE
Intermittent failures in tests

### ISSUE THREAD

**Issue by berfr** [jsvine/markovify#134]
There seems to be two types of failures happening in the tests. They are as follows:

As seen in https://travis-ci.org/github/jsvine/markovify/jobs/699302439:

```
Traceback (most recent call last):
  File "/home/travis/build/jsvine/markovify/test/test_basic.py", line 39, in test_chain
    sent = new_text_model.make_sentence()
  File "/home/travis/build/jsvine/markovify/markovify/text.py", line 215, in make_sentence
    if max_words != None and len(words) > max_words or len(words) < min_words:
TypeError: '<' not supported between instances of 'int' and 'NoneType'
```

As seen in https://travis-ci.org/github/jsvine/markovify/jobs/633117450:

```
Traceback (most recent call last):
  File "/home/travis/build/jsvine/markovify/test/test_basic.py", line 108, in test_make_sentence_with_start_three_words
    assert(markovify.chain.BEGIN not in sent)
TypeError: argument of type 'NoneType' is not iterable
```

Both errors happen in any type of test configuration and seem to be caused by `model.make_sentence()` calls or what is or is not returned.

**Comment by jsvine** [comment]
Thanks for flagging these, @berfr!

The first issue is, I believe, was a very brief one introduced in v0.8.1 and fixed in v0.8.2. But let me know if you're still running into that error in v0.8.2+.

The second, I believe, is due to the inherently random nature of Markov chains. Every once in a while, a test fails because the code in that test cannot produce a complete chain in X `tries`. One possibility for fixing this would be to set [random.seed()](https://docs.python.org/3/library/random.html#random.seed). Or do you diagnose the issue/solution differently?

**Comment by berfr** [comment]
Ok, that makes sense. I haven't looked into the source or solution yet but would be happy to, eventually. I'll comment here again when I have more info!

**Comment by berfr** [comment]
For reference, the intermittent failing test output had become:

```
Traceback (most recent call last):
  File "/home/travis/build/jsvine/markovify/test/test_basic.py", line 109, in test_make_sentence_with_start_three_words
    sent = text_model.make_sentence_with_start("Sherlock")
  File "/home/travis/build/jsvine/markovify/markovify/text.py", line 275, in make_sentence_with_start
    raise ParamError(err_msg)
markovify.text.ParamError: `make_sentence_with_start` can't find sentence beginning with Sherlock
```

since #132 as seen [here](https://travis-ci.org/github/jsvine/markovify/jobs/701879556).

The test should be fixed with #137.
